pages/Microcopy.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { ProjectPageHeading } from './components/ProjectPageHeading';
+
+export function Microcopy() {
+  return (
+    <main>
+      <h1>Microcopy</h1>
+
+      <p>
+        Copy refers to text that is used on the website. Microcopy refers to
+        small pieces of text on the website such as button or navigation labels,
+        help text, popover text and so on. Microcopy uses compressed syntax and
```

Comment:
Not important, and if you think it's covered that's fine, but I would explicitly mention somehow the error texts, a use case we are dealing with all the time.